spec/unit/mutant/subject/method/instance_spec.rb

```diff
@@ -137,7 +153,7 @@ def foo; end
     it_should_behave_like 'a command method'
   end
 
-  describe '#mutations', mutant_expression: 'Mutant::Subject#mutations' do
+  describe '#mutations' do
```

Comment:
@snusnu The fix for the last alive mutation was removing this overly specific overwrite!